src/lib/component/base/Drawer/Drawer.tsx

```diff
@@ -101,7 +101,12 @@ const Drawer = ({
       overflow="hidden"
     >
       <Box display="flex" alignItems="center" height={heights.toolbar}>
-        <IconButton color="inherit" onClick={onToggle}>
+        <IconButton
+          color="inherit"
+          onClick={onToggle}
+          aria-label="main-menu"
```

Comment:
Should `main-menu` be `Main menu` ? Just noticing that you have spaces in other areas of the code

Reply:
good callout